src/client/ContentManagement.js

```diff
@@ -154,12 +154,12 @@ exports.SetPermission = async function({objectId, permission, writeToken}) {
  * @returns {Promise<string>} - Object ID of created content type
  */
 exports.CreateContentType = async function({name, metadata={}, bitcode}) {
-  let tenantContractId = this.userProfileClient.TenantContractId();
+  let tenantContractId = await this.userProfileClient.TenantContractId();
   if (!tenantContractId) {
-    throw(e, "ERROR: Can not find metadata _ELV_TENANT_ID that belongs to this user");
+    throw("ERROR: Can not find metadata tenantContractId that belongs to this user");
```

Comment:
Let's change this error message to say:

`User address is not associated with a tenant`